test/functional/editions_controller_test.rb

```diff
@@ -39,6 +39,53 @@ class EditionsControllerTest < ActionController::TestCase
     end
   end
 
+  context "#request_amendments" do
+    should "render the 'Request amendments' page" do
+      get :request_amendments_page, params: { id: @edition.id }
+      assert_template "secondary_nav_tabs/request_amendments_page"
+    end
+  end
+
+  context "#request_amends" do
```

Comment:
Context needs updating now that we've renamed the routes and controller methods.
```suggestion
  context "#request_amendments" do
```